src/main/java/com/facebook/openwifirrm/modules/ModelerUtils.java

```diff
@@ -236,32 +237,59 @@ private static boolean matchesForAggregation(WifiScanEntry entry1, WifiScanEntry
 	}
 
 	/**
-	 * For each AP, for each other AP that sent a wifiscan entry to that AP, this
-	 * method calculates an aggregate wifiscan entry with an aggregated RSSI.
+	 * For each AP, for each other AP that sent a wifiscan entry to that AP,
+	 * this method calculates an aggregate wifiscan entry with an aggregated
+	 * RSSI. If no non-obsolete entry exists, the latest wifiscan entry is used
+	 * instead.
 	 *
-	 * @param dataModel          the data model which includes the latest wifiscan
-	 *                           entries
-	 * @param obsoletionPeriodMs for each (scanning AP, responding AP) tuple, the
-	 *                           maximum amount of time (in milliseconds) it is
-	 *                           worth aggregating over, starting from the most
-	 *                           recent scan entry for that tuple, and working
-	 *                           backwards in time. An entry exactly
-	 *                           {@code obsoletionPeriodMs} ms earlier than the most
-	 *                           recent entry is considered non-obsolete (i.e., the
-	 *                           "non-obsolete" window is inclusive). Must be
-	 *                           non-negative.
+	 * @param dataModel          the data model which includes the latest
+	 *                           wifiscan entries
```

Comment:
I accidentally used my eclipse formatter.  It's ok, since Jeffrey is going to format the repo soon anyways, I won't worry about format too much in this PR.

Reply:
The autoformatter is not going to touch comments, so format them yourself if needed (I don't really care as long as it's readable)